packages/wallets/src/components/Base/WalletTextField/WalletTextField.tsx

```diff
@@ -53,7 +52,9 @@ const WalletTextField = forwardRef<HTMLInputElement, WalletTextFieldProps>(
                             {label}
                         </label>
                     )}
-                    <div className='wallets-textfield__icon'>{renderRightIcon?.()}</div>
+                    {typeof renderRightIcon === 'function' && (
```

Comment:
small one: is typeof needed? as a rule of thumb, usage of typeof is a code smell (in this case, very minor), would  just {renderRightIcon && (<div ...>...</div>)} be sufficient?

Reply:
This was implemented in response to another review (https://github.com/binary-com/deriv-app/pull/11402#discussion_r1392268090) as renderRightIcon is a function and couldn't be used as a checking condition